src/DurableTask.Netherite.AzureFunctions/NetheriteProvider.cs

```diff
@@ -205,14 +205,16 @@ ScaleStatus GetScaleStatusCore(int workerCount, NetheriteScaleMetrics[] metrics)
             {
                 ScaleRecommendation recommendation;               
                 try
-                { 
-                    if (metrics == null || metrics.Length == 0)
```

Comment:
is this expected behavior, or just something that we know is happening transiently and that we do not want to fail catastrophically?

If it's expected, then LGTM (and will approve as such). However, if it's not expected, should we log something?

Reply:
I am not aware of any specification for the scale controller API so I can't tell whether it is expected or not. But it does happen transiently.

As with other scale controller abnormalities, it is not clear to me what would be the best way to log/report them. We are already logging the "missing metrics" description as part of the scale decision, so it is not completely invisible. We could also generate a warning.